nomos-da/full-replication/src/lib.rs

```diff
@@ -188,7 +193,10 @@ impl DaProtocol for FullReplication<AbsoluteNumber<Attestation, Certificate>> {
     type Settings = Settings;
 
     fn new(settings: Self::Settings) -> Self {
-        Self::new(AbsoluteNumber::new(settings.num_attestations))
+        Self::new(
```

Comment:
one thing I was thinking is that authentication of the sender is probably orthogonal to the da protocol and we might want to keep them separate. Anyway, for now we don't do any real authentication so this is ok